packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.less

```diff
@@ -7,14 +7,13 @@
     }
   }
 
-  /* stylelint-disable selector-type-no-unknown */
+  /* stylelint-disable-next-line selector-type-no-unknown */
```

Comment:
Still needed because it doesn't suit with: https://stylelint.io/user-guide/rules/selector-type-no-unknown#ignore-custom-elements-default-namespace